client/src/components/USBController.tsx

```diff
@@ -21,6 +21,10 @@ const USBController = () => {
     return <Text>Connecting…</Text>
   }
 
+  if (status === UsbDriveStatus.recentlyEjected) {
+    return <Text>Ejected</Text>
```

Comment:
This component should always return a button. 
```suggestion
    return <Button small disabled onPress={() => {}}>Ejected</Button>
```

Reply:
Why? It's not actionable.

Reply:
This button is displayed along with other buttons in a navigation which is expecting buttons.

If this element changes from a `<button>` tag to a `<p>` tag when not expected, then things may not be formatted as expected. 

A disabled button is also "not actionable" 😉